app/helpers/stacks_helper.rb

```diff
@@ -26,4 +38,24 @@ def pull_request_link(commit)
   def render_raw_commit_id_link(commit)
     link_to(commit.short_sha, github_commit_url(commit), target: '_blank', class: 'number')
   end
+
+  private
+
+  def deploy_button_disabled?(commit)
```

Comment:
I think `deploy_button_disabled?` fits more with this case. Usually I use `can_\w+?` something when we method receiver is an actor, like a user, in this case the receiver is the helper and it is related to Stack that is not the actor of the action.